docs/tutorials/semantic_search/asymmetric_embedding_model.md

```diff
@@ -0,0 +1,541 @@
+# Tutorial: Running Asymmetric Semnantic Search within OpenSearch
+
+This tutorial demonstrates how generating text embeddings using an asymmetric embedding model in OpenSearch. The embeddings will be used
+to run semantic search, implemented using a Docker container. The example model used in this tutorial is the multilingual
+`intfloat/multilingual-e5-small` model from Hugging Face. 
+You will learn how to prepare the model, register it in OpenSearch, and run inference to generate embeddings.
+
+> **Note**: Make sure to replace all placeholders (e.g., `your_`) with your specific values.
+
+---
+
+## Prerequisites
+
+- Docker Desktop installed and running on your local machine.
+- Basic familiarity with Docker and OpenSearch.
+- Access to the Hugging Face `intfloat/multilingual-e5-small` model (or another model of your choice).
+---
+
+## Step 1: Spin up a Docker OpenSearch cluster
+
+To run OpenSearch in a local development environment, you can use Docker and a preconfigured `docker-compose` file.
+
+### a. Create a Docker Compose File
+
+You can use this sample [file](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/docker/#sample-docker-compose-file-for-development) as an example.
+Once your `docker-compose.yml` file is created, run the following command to start OpenSearch in the background:
+
+```
+docker-compose up -d
+```
```

Comment:
I think this part is bit unnecessary. We can assume customer already setup their cluster and then come here to setup a model?

Reply:
I understand your view point I figured it would be nice to provide a tutorial with a docker setup since there aren't many of them in our repo. Although, we can introduce the header with something along the lines of You can skip step 1 if you want to do it your own way.

Reply:
I agree that we don't have any docker setup tutorial in our repo, but I think that responsibility lies in Core. May be we can add url for opensearch setup in general. My worry that, customer might think that they need docker setup for asynmmetric models, which is not the case.